### ISSUE TITLE
Milestone 1 Feedback

### ISSUE THREAD

**Issue by wddlz** [nforesti/kiddos#1]
Good population and idea, but prototype 2 might become a little complicated for an elderly person who wants a simple way to find a device, both in the granularity of your filters and potentially the reviews. For example, if one of the reviews is from Marques Brownlee, it might be way too complicated to present to the user, and so, you may need a way to filter these reviews on simplicity. Prototype 2.3 image, is a good and simple UI compared to some (like 2.2 image) that throw a lot of data at someone who is unlikely to want to sift through it all.

Rubric items fulfilled: [1,2,3,4,5,6,7,8,9,10,11,12,13] (13/13)

You are in: Group A for grading (check schedule for what days you need to show up to lecture)
Your TA is: Ian

**Comment by shira462** [comment]
Hi Ian, thank you for your feedback. Below is our alternative project idea, since we found problems with the data-set for our original idea. We would like feedback as to whether the API and Dataset listed below satisfy the project requirements, and any suggestions you might have. 

Population: Nonverbal Autistic Children

Proposal #2: A web-app ideally hosted on a tablet or Ipad to aid nonverbal autistic children communicate through an icon or image-based interface. 

API: Google Cloud Text-to-Speech (upon clicking on a particular image with subtext)

Dataset: Icons & Images database such as https://www.flaticon.com/

Paper Prototype (click on image to view right side up) 
![20190423_114625](https://user-images.githubusercontent.com/38950258/56699038-cc525f00-66a8-11e9-9a03-5e23a4cf44d9.jpg)

**Comment by wddlz** [comment]
Yep, this is a very interesting idea (and a population who likely needs more designers working to help them). I don't know enough about this area to give feedback on viability (for example, is presenting them with an icon of a male figure enough for "dad" or do they need an actual image of dad, and in that case you'd need to upload images or let the user decide). You may also have to curate your dataset since not all icon packs are good designs, especially for children. 

You also have navigation by selecting category -> specific thing within category -> perhaps even more specific thing within category. I wonder if you can leverage autocomplete functionality to assist the user. For example, user selects "I want" (whatever icon that would be, see what I mean about icons possibly being ambiguous design), the next set of icons should just be objects that the user could want (e.g. some water, food, to go to the bathroom, etc.). This might increase the level of expression you afford to the user, and would move beyond just a digital "see'n'say"
